src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -101,18 +102,42 @@ const EnableWhenCondition: React.FC<EnableWhenConditionProps> = ({
     name: `${enableWhenPath}.${index}`,
   });
 
+  const {
+    field: { onChange: onChangeOperator },
+  } = useController({
+    control,
+    name: `${enableWhenPath}.${index}.operator`,
+  });
+
   const dependentItem = useMemo(
     () => (state?.question ? itemMap[state?.question] : undefined),
     [itemMap, state?.question]
   );
+
```

Comment:
Found an unrelated bug while testing this, but it looks like its on release-129 too. Added a separate issue: https://github.com/open-path/Green-River/issues/6512